### ISSUE TITLE
Does not work in RStudio viewer panel

### ISSUE THREAD

**Issue by Marlin-Na** [freestatman/ideogRam#5]
It seems that the widget does not work in RStudio viewer, which may due to the fact that its browser engine does not currently support some new language features of JS, which ideogram.js might used.

The web console logged the following error:
```
SyntaxError: Unexpected token ')'                                       ideogram.min.js:9
```
  
RStudio Version 1.1.383
Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/538.1 (KHTML, like Gecko) RStudio Safari/538.1 Qt/5.4.0

**Comment by eweitz** [comment]
@Marlin-Na, I suspect RStudio doesn't parse newer features in JavaScript, i.e. ES6.  Can you try again with the just-released version of Ideogram, [v0.14.0](https://github.com/eweitz/ideogram/releases/tag/v0.14.0)?

**Comment by Marlin-Na** [comment]
Hi @eweitz , thanks! The new version is now working with Rstudio panel.

However, with the new version, the web console of Rstudio still has the error: `SyntaxError: Unexpected token ')'` **without** an indication on line number, i.e. `ideogram.min.js:9` like the previous one.
But everything seem to work fine.

Also, in the repo, I have found a [css file](https://github.com/Marlin-Na/ideogRam/blob/master/inst/htmlwidgets/lib/ideogram/ideogram.css), I am not sure where is it from as I did not find it in the `dist` directory of ideogram (possibly from a previous version of ideogram?).

I have tried to remove the css file, finding that without the css file, the widget will look ugly in Rstudio viewer panel (see below) but normal in Chrome. I am not sure if this is some kind of clue to the `SyntaxError: Unexpected token ')'` error above.

![export-ideogram-withoutcss-inrstudiopanel](https://user-images.githubusercontent.com/9980776/35138557-1edf605e-fd2a-11e7-88f9-44b3aeda810b.jpeg)

Anyway these are minor problems as it is already working in RStudio viewer panel.
